content/docs/concepts/secrets/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Secret Service
+description: Learn how to manage secrets in a Keptn cluster.
+weight: 20
+keywords: [keptn, use-cases]
+---
+
+The **Secret Service** is used to manage secrets in a Keptn Cluster.
```

Comment:
Add version where it was introduced?

Reply:
Excellent idea!  How do I find that information?

Reply:
0.8.1 according to https://github.com/keptn/keptn/pull/3595